tests/api2/jail.py

```diff
@@ -326,11 +280,21 @@ def test_20_verify_jail_stopped():
             break
 
 
-def test_21_rc_action():
+def test_19_rc_action():
     results = POST('/jail/rc_action/', 'STOP')
     assert results.status_code == 200, results.text
 
 
+def test_20_delete_jail():
+    results = DELETE(f'/jail/id/{JAIL_NAME}/')
```

Comment:
Let's the `force` flag here too please. If a jail is running and we intend to destroy it without stopping and without providing `force`, it will fail whereas if it's running and `force` is provided it shouldn't fail and destroy the jail.